client/src/app/game/game-page/game-page.component.ts

```diff
@@ -11,7 +11,7 @@ export class GamePageComponent implements OnInit {
 
   response : any = {};
   image : string = "";
-
+  ratingData : any;
```

Comment:
I would say it's better to set this to `number`, as typelessness can be devious.

Reply:
`ERROR in src/app/game/game-page/game-page.component.ts(36,7): error TS2322: Type 'Object' is not assignable to type 'number'.`

Reply:
ratingData is an Object, use that instead